src/main/java/com/arpnetworking/tsdcore/sinks/WavefrontSink.java

```diff
@@ -66,9 +67,25 @@ public void recordAggregateData(final PeriodicData periodicData) {
             // Build the list of annotations based on cluster and white listed dimensions
             final Map<String, String> annotations = Maps.newHashMap();
             annotations.put("cluster", fqdsn.getCluster());
```

Comment:
Ah but I didn't want to change the subtle semantics of choosing `cluster` from the `fqdsn` vs. from the dimensions. If you tell me that choosing from the dimensions is equivalent, I'll believe you.